packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -633,9 +618,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
             const operationMode = getOperationMode();
             if (operationMode === 'ERROR') {
                 // eslint-disable-next-line max-len
-                await this.setStatusMessage(`Experiencing problems, ${this.stats.state.requestsFailed - previousState.requestsFailed || this.stats.state.requestsFailed} errors in the past ${this.loggingInterval} seconds.`,
```

Comment:
i wasnt sure about this part, we could also use `warningOnce`, but due to the dynamic nature of the message, i guess it would be printed way too often anyway, so decided to keep it all on `debug` level only